docs/cookbook.md

```diff
@@ -460,6 +461,36 @@ implicit class RichSFuture[A](val f: SFuture[A]) extends AnyVal {
 }
 ```
 
+### Server Sent Events
+
+Finch has basic support for Server Sent Event using `AsyncStream` that has a content type of `Text.EventStream`.
+```scala
+import com.twitter.concurrent.AsyncStream
+import com.twitter.finagle.http.Http
+import io.finch._
+
+case class Example(x: Int)
+def exampleStream: AsyncStream[Example] = ???
+
+implicit def encodeEventStream[A <: AnyRef]: Encode.EventStream[AsyncStream[A]] = ???
```

Comment:
I don't think users need to provide that. Given that there is `Show[Example]`, everything should just work, right?